subgraphs/uniswap-v3/src/common/price/price.ts

```diff
@@ -140,7 +140,7 @@ export function findUSDPricePerToken(
       let poolAmounts = getLiquidityPoolAmounts(poolAddress);
       let pool = getLiquidityPool(poolAddress);
 
-      if (pool.outputTokenSupply!.gt(BIGINT_ZERO)) {
+      if (pool.totalValueLockedUSD!.gt(BIGDECIMAL_ZERO)) {
```

Comment:
Is this change safe?
I understand this change is because we are no longer tracking the output token, so need to change the condition. But would it behave as expected if the pool consists of 2 non-whitelisted tokens? or the totalValueLockedUSD is 0 for some other reason (low liquidity, etc)?

Reply:
There are still checks for minimum liquidity for the paired token, so I still think it is a safe change. All this this condition was was a sanity check that there were any liquidity positions, but I think it was unnecessary, unless used for error logs. 

Actually on second thought though, checking tvl might not even be necessary since it still asks that the value of the paired token is greater than the minimum liquidity threshold.